frontend/app/actions/ActionRefiners.scala

```diff
@@ -93,10 +93,10 @@ class ActionRefiners(authenticationService: AuthenticationService, parser: BodyP
 // this is helping us stack future/either/option
 object OptionEither {
 
-  type FutureEither[X] = EitherT[Future, String, X]
+  type FutureEither[X] = EitherT[String, Future, X]
```

Comment:
scalaz has swapped the order of type parameters for EitherT

Reply:
how helpful 😆 
so this is now the opposite of cats' `EitherT`